<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmployeeInfo</class>
 <widget class="QDialog" name="EmployeeInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>271</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>Info</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Eid</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_eid</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_eid"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Name</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_name</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_name"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Surname</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_surname</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_surname"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Age</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_age</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_age"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_sec_status">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>370</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>361</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_loadTable">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>300</y>
     <width>201</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Table</string>
   </property>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>250</y>
     <width>181</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_loadList">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>460</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Load List</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>360</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_comboBox">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>440</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Combo</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
